docs/API.md

```diff
@@ -557,32 +673,49 @@ context than inside the listener callback handler, the value is undefined.
 
 ---
 
-| OptName               | Since | Binding | Type   | Units  | Default  | Range  |
-| --------------------- | ----- | ------- | ------ | ------ | -------- | ------ |
-| `SRTO_GROUPSTABTIMEO` |       | pre     | `int`  | ms     | 40       | 10+    |
+| OptName               | Since | Binding | Type       | Units  | Default  | Range  | Dir | Entity |
+| --------------------- | ----- | ------- | ---------- | ------ | -------- | ------ | --- | ------ |
+| `SRTO_GROUPSTABTIMEO` |       | pre     | `int32_t`  | ms     | 40       | 10-... | W   | GSD+   |
 
 - This setting is used for groups of type `SRT_GTYPE_BACKUP`. It defines the stability 
 timeout, which is the maximum interval between two consecutive packets retrieved from 
-the peer on the currently active link.
-  
-- NOTE: The two packets can be of any type, but this setting usually refers to control
-packets while the agent is a sender. Idle links exchange only keepalive messages once 
-per second, so they do not count. 
-
-- The value of `SRTO_GROUPSTABTIMEO` should be set as small as possible. Keep
-in mind that the time between two consecutive ACK messages (default = 10 ms)
-will vary, hence 40ms is the recommended setting. Note that higher values
-increase the latency penalty. When this time is exceeded, and the link is
-therefore considered unstable, one of the idle links gets activated, at which
-point all packets unacknowledged so far are sent first.
+the peer on the currently active link. These two packets can be of any type,
+but this setting usually refers to control packets while the agent is a sender.
+Idle links exchange only keepalive messages once per second, so they do not
+count. Note that this option is meaningless on sockets that are not members of
+the Backup-type group.
+
+- This value should be set with a thoroughly selected balance and correspond to
+the maximum stretched response time between two consecutive ACK messages. By default
+ACK messages are sent every 10ms (so this interval is not dependent on RTT at all),
+and so should be the distance between two consecutive received ACK messages,
+however the network jitter in public internet causes these intervals to be
+stretched even to multiplicity of that interval. Both great and little values
+of this option have consequences:
+
+- Great value of this option prevents overreaction on highly streteched response
+times, but this value comprises a latency penalty - the latency must be greater
+than this value or otherwise switching to another link won't be able to preserve
+smooth signal sending. High values will also contribute to higher burst of
+packets sent at the moment when an idle link is activated.
```

Comment:
```suggestion
- Large values of this option prevent overreaction on highly stretched response
times, but introduce a latency penalty - the latency must be greater
than this value (otherwise switching to another link won't preserve
smooth signal sending). Large values will also contribute to higher 
packet bursts sent at the moment when an idle link is activated.
```